tests/syntax/test_regions.py

```diff
@@ -57,14 +56,14 @@ def test_sector_lazy():
     ego = sampleEgoFrom("""
         vf = VectorField("Foo", lambda pos: 2 * pos.x)
         x = 0 relative to vf
-        ego = Object at Range(-1, 1) @ 0, with foo SectorRegion(0@0, x, 0, 45 deg)
+        ego = new Object at Range(0, 1) @ 0, with foo SectorRegion(0@0, x.yaw, 0, 45 deg)
     """)
-    assert ego.foo.radius == 2 * ego.position.x
+    assert ego.foo.radius == pytest.approx(2 * ego.position.x, abs=1e-6)
```

Comment:
Is it necessary to set the absolute precision here? I wouldn't have thought we would lose so much precision going through a quaternion and back.